src/Microsoft.Health.Dicom.SqlServer/Features/Store/SqlIndexDataStore.cs

```diff
@@ -250,13 +256,26 @@ private async Task DeleteInstanceAsync(string studyInstanceUid, string seriesIns
             using (SqlConnectionWrapper sqlConnectionWrapper = await _sqlConnectionFactoryWrapper.ObtainSqlConnectionWrapperAsync(cancellationToken))
             using (SqlCommandWrapper sqlCommandWrapper = sqlConnectionWrapper.CreateSqlCommand())
             {
-                VLatest.DeleteInstance.PopulateCommand(
+                if (_schemaInformation.Current >= SchemaVersionConstants.SupportCustomTagSchemaVersion)
```

Comment:
Why do we need this? The prc signature is not changing on different versions.

Please understand the reason on why. I am happy to explain

Reply:
Realized why this is unnecessary!! Reverted this change.